CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 4.1.17
+  - Added configuration setting `check_archive_validity` settings to enable gzipped files verification, 
+  issue [#261](https://github.com/logstash-plugins/logstash-input-file/issues/261)
+## 4.1.16
+  - Added configuration setting `check_archive_validity` settings to enable gzipped files verification, 
+  issue [#261](https://github.com/logstash-plugins/logstash-input-file/issues/261)
+
```

Comment:
Two comments about the changelog: 
* It looks like the changelog contains the same info for both 4.1.16 and 4.1.17. 
* I had simultaneiously claimed 4.1.17 for changes in #266. Let's coordinate on version bump and publishing.

Reply:
The doubling of description was my fault on conflict resolution, now fixed. For the the rest, we could merge your `4.1.17' and this one so that we publish just one time, WDYT?